samples/TR/W3C.css

```diff
@@ -12,13 +12,13 @@ body {
 	padding: 0 !important;
 }
 @page:left {
-	margin: 10% 10% 10% 14%!important;
+	margin: 10% 10% 10% 10% !important;
```

Comment:
Right, it is a matter of taste. I think centered page numbers might be better, but it is subjective, so if you prefer to keep it this way, that's ok with me.